core/interfaces.go

```diff
@@ -111,6 +111,8 @@ type StorageGetter interface {
 	GetCertificateByShortSerial(string) (Certificate, error)
 	GetCertificateStatus(string) (CertificateStatus, error)
 	AlreadyDeniedCSR([]string) (bool, error)
+	GetSCTReceipts(string) ([]SignedCertificateTimestamp, error)
```

Comment:
As far as I can tell, the `GetSCTReceipt(s)` methods are only used in tests so far. To reduce the scope of this change, can you pull them out for now? We'll probably want to add them again later, so hang on to a copy of the branch, but I generally want to avoid adding code before it is in use.